Formula/xdk.rb

```diff
@@ -0,0 +1,23 @@
+class Xdk < Formula
+  desc "Ecstasy Development Kit (XDK)"
+  homepage "https://github.com/xtclang/xvm/"
+  url "https://github.com/xtclang/xvm/releases/download/v0.4.2/xdk-0.4.2.tar.gz"
+  sha256 "b0b5217abf05de986e6211a7c8665b304fa89df1726749e4f1dd7eff1cbbe1fe"
+  license "Apache-2.0"
+
+  depends_on "openjdk"
+
+  def install
+    libexec.install buildpath.children
+    %w[xec xtc xam].each do |cmd|
+      java = "${JAVA_HOME}/bin/java"
+      args = "-Xms256m -Xmx1024m -ea -jar #{libexec}/javatools/javatools.jar" \
+             " #{cmd} -L #{libexec}/lib -L #{libexec}/javatools"
+      (bin/cmd).write_env_script java, args, Language::Java.overridable_java_home_env
+    end
+  end
+
+  test do
+    system "#{bin}/xec", "-help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.